EIPS/eip-5069.md

```diff
@@ -1,44 +1,131 @@
+ - **Publishing Proposals**: Making proposals, including their history and associated discussions available over the long term at no cost.
+    
+    By doing so, we foster transparency and ensure that valuable insights from past proposals are accessible for future decision-making and learning.
+ - **Facilitating Discussion**: Providing a forum for discussing proposals open to anyone who wants to participate civilly.
+    
+    By encouraging open dialogue and collaboration, we aim to harness the collective knowledge and expertise of the Ethereum community in shaping proposals.
 
-- Good communication skills
-- Ability to handle contentious discourse
-- 1-5 spare hours per week
+ - **Upholding Quality**: Upholding a measure of minimally-subjective quality for each proposal as defined by its target audience.
 
-The best available resource to understand the EIP process is [EIP-1](./eip-1.md). Anyone desirous of becoming an EIP editor MUST understand this document. Afterwards, participating in the EIP process by commenting on and suggesting improvements to PRs and issues will familliarize the procedure, and is recommended. The contributions of newer editors should be monitored by other EIP editors.
+    By adhering to defined criteria, we promote the development of high-quality and relevant proposals that drive the evolution of Ethereum.
+    
+### What we Don't
 
-Anyone meeting the above requirements may make a pull request adding themselves as an EIP editor and adding themselves to the editor list at `config/eip-editors.yml` and in [EIP-1](./eip-1.md). If every existing EIP editor approves, the author becomes a full EIP editor. This should notify the editor of relevant new proposals submitted in the EIPs repository, and they should review and merge those pull requests.
+On the other hand, we do _not_:
 
-### Special Merging Rules for this EIP
+ - **Decide Winners**: If there are multiple competing proposals, we will publish all of them. We are not in the business of deciding what is the right path for Ethereum, nor do we believe that there is One True Way to satisfy a need.
 
-This EIP MUST have the same rules regarding changes as [EIP-1](./eip-1.md).
+ - **Assert Correctness**: While we might offer technical feedback from time to time, we are not experts nor do we vet every proposal in depth. Publishing a proposal is not an endorsement or a statement of technical soundness.
 
-## Rationale
+ - **Manage**: We do not track implementation status, schedule work, or set fork dates or contents.
 
-- "6 months" was chosen as the cutoff for denoting `Stagnant` EIPs terminally-`Stagnant` arbitrarily.
-- This EIP requires special merging rules for the same reason [EIP-1](./eip-1.md) does.
+ - **Track Registries**: We want all proposals to eventually become immutable, but a registry will never get there if anyone can keep adding items. To be clear, exhaustive and/or static lists are fine. 
+
```

Comment:
should we also add to DONOTs:

` - track or evaluate copyrights or prior art or credits`

Reply:
```suggestion
- **Track Intellectual Property Claims**: Trademarks, copyrights, and patents are the responsibility of document authors and implementers, not the editors of the EIP process.
- **Track or evaluate Prior Art**: While editors may anecdotally comment on prior art in or outside of the EIP and EVM corpus, there is no expectation or assumption that they will evaluate novelty, assist in prior art review, or track contributions or acknowledgements.

```

Reply:
How about we lump it all into one bullet point:

```suggestion
 - **Provide Legal Advice**: Trademarks, copyrights, patents, prior art, and other legal matters are the responsibility of authors and implementers, not EIP Editors. We are not lawyers, and while we may occasionally make comments touching on these areas, we cannot guarantee any measure of correctness.
```

Reply:
i am fine with either of the versions :+1: